keda/redis-hpa.yaml

```
@@ -10,10 +10,10 @@ spec:
   pollingInterval: 3  # Optional. Default: 30 seconds
   cooldownPeriod:  5 # Optional. Default: 300 seconds
   scaleTargetRef:
-    deploymentName: dummy
+    name: dummy
   triggers:
     - type: redis
       metadata:
-        address: REDIS_ADDRESS
+        addressFromEnv: REDIS_ADDRESS
         listName: default
         listLength: "10"
```

```suggestion
        listLength: "10"

```
Let's add blank new line

---

@turbaszek Do you mean adding a new line between listLength and listName? or between every metadata field? Also should we Add this line in every file in keda/ folder?

---

@Ritikaa96 I mean adding new line at the end of the file